src/test/e2e/stores/client-application-store.e2e.test.ts

```diff
@@ -125,15 +132,18 @@ test('Merge keeps value for single row in database', async () => {
         appName: clientRegistration.appName,
         description: 'new description',
     });
-    const stored = await clientApplicationsStore.getApplication(
+    const stored = await clientApplicationsStore.get(
```

Comment:
getApplication and get methods are the same, should we remove one of them?

Reply:
getApplication doesn't exist on the store interface. it was written only on the fake store, but yes, we could remove it.

Reply:
https://github.com/Unleash/unleash/blob/e27a578fab648c395f80fccc5d59636d3704db21/src/lib/db/client-applications-store.ts#L157-L169 isn't it this one?